analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -39,28 +39,38 @@ spanningFragCountFilter=100
 
 
 # relevant gene expression files
-polya_expression_file="${data_path}/pbta-gene-expression-rsem-fpkm.polya.rds"
-stranded_expression_file="${data_path}/pbta-gene-expression-rsem-fpkm.stranded.rds"
+rna_expression_file="${data_path}/gene-expression-rsem-tpm-collapsed.rds"
 
 # reference expression file
-normal_expression_file="${references_path}/Brain_FPKM_hg38_matrix.txt.zip"
+normal_expression_adrenal_gland="${references_path}/gtex_adrenal_gland_TPM_hg38.rds"
+normal_expression_brain="${references_path}/gtex_brain_TPM_hg38.rds"
 
 # metadata files
-if [[ RUN_FOR_SUBTYPING == "0" ]]
+if [[ RUN_FOR_SUBTYPING -eq "0" ]]
```

Comment:
Is there an advantage over using `-eq` instead of `==`? just curious!

Reply:
Not really - just to match what we have been using for independent samples - but not necessary at all.